### DISCUSSION TITLE
Keep session-based history of played beatmaps

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#17580] [Ideas] [answered]
Game could hold for one session a history of plays, and possibly of selected beatmaps at beatmap select (but didn't click play). This would help to navigate in big libraries of beatmaps and e.g. getting back to something you want to practice once more. One possibilty of UI is to put it as an overlay coming from the clock at the top bar of the game

**Comment by peppy** [comment, marked as answer]
Recently played beatmaps is a sort order at song select.
